### ISSUE TITLE
use `@typescript-eslint/parser` with `@angular-eslint/template-parser` in angular template

### ISSUE THREAD

**Issue by xiaoxiangmoe** [angular-eslint/angular-eslint#33]
It will be better if we have documents for combine `@typescript-eslint/parser` with `@angular-eslint/template-parser` in angular template.

---

for example, [vue-eslint-parser:  parserOptions.parser](https://github.com/mysticatea/vue-eslint-parser#parseroptionsparser):

```js
{
    "parser": "vue-eslint-parser",
    "parserOptions": {
        "parser": "@typescript-eslint/parser"
    }
}
```

**Comment by xiaoxiangmoe** [comment]
@JamesHenry Is there any documentation? Thank you very much.

**Comment by BO41** [comment]
Probably not yet.
But if you have any understanding about this, **please** feel free to **open a PR** ;)

Seems that `@angular-eslint/template-parser` is not involving `@typescript-eslint/parser` which will bring many people problems

**Comment by xiaoxiangmoe** [comment]
Can we lint template string in `foo.component.ts`?

```ts
@Component({
  selector: 'app-foo',
  template: `
    <div>lint template here</div>
  `,
})
export class FooComponent {}
```

**Comment by BO41** [comment]
> Can we lint template string in `foo.component.ts`?

I think https://github.com/angular-eslint/angular-eslint/pull/39 added a lint checking inline tempaltes

**Comment by BO41** [comment]
> @JamesHenry Is there any documentation?

accoring to https://github.com/angular-eslint/angular-eslint/pull/34#issuecomment-586315326 this commit https://github.com/angular-eslint/angular-eslint/pull/38 adds "documentation"

**Comment by JamesHenry** [comment]
I will focus on the inline template story next, there were other things to figure out in the template-parser first and those are now done as of today

**Comment by JamesHenry** [comment]
In the meantime, you can see how to use the two parsers together with .html template files in the integration test in this repo

**Comment by JamesHenry** [comment]
#47 addressed this, and the changes are available in `v0.0.1-alpha.21` of the packages. Please check out the PR and the integration test configuration for how to configure template linting together with regular linting of your source code.

We are getting very close to a first stable release now! (And from that point there will be proper automated releases with changelogs etc)
